parse.go

```diff
@@ -263,7 +264,7 @@ func cmdFromStruct(name string, dest path, t reflect.Type) (*command, error) {
 	walkFields(t, func(field reflect.StructField, t reflect.Type) bool {
 		// Check for the ignore switch in the tag
 		tag := field.Tag.Get("arg")
-		if tag == "-" {
+		if tag == "-" || (unicode.IsLetter(rune(field.Name[0])) && unicode.IsLower(rune(field.Name[0]))) {
```

Comment:
Would you add a check that `len(field.Name) > 0`?

And could this be factored this out into a function named `isExported`?

Reply:
I have done as reccomended but with an `isUnexportedField` function.